migrations/app/20171005090919_app_workload_add_arms_columnms.js

```diff
@@ -0,0 +1,18 @@
+exports.up = function (knex, promise) {
+  var sql = 'ALTER TABLE app.workload ADD arms_community_cases INT NOT NULL CONSTRAINT workload_arms_community_default DEFAULT 0;'
```

Comment:
Purposely leaving adding the new column to WPC until another PR?

Reply:
Yea, I was because I think it's easier to stay on top of if we keep PRs to the smallest logical change, that will still build. Happy to reassess this position though.